tanner/tests/test_base.py

```diff
@@ -124,3 +124,26 @@ def test_set_injectable_page(self):
             sess = session.Session(None)
         injectable_page = self.handler.set_injectable_page(sess)
         self.assertEqual(injectable_page, '/python.html')
+
+    def test_emulate(self):
+        data = dict(method='GET', path='/path.html?file=/etc/passwd',
+                    cookies={'sess_uuid': '9f82e5d0e6b64047bba996222d45e72c'})
+
+        async def mock_lfi_handle(attack_value, session):
```

Comment:
To test a certain method, you can mock other methods as they will have their tests separately  so no need for all this just mock `handle_get`

Reply:
You need to check for different results from `handle_get` will you get correct `type`, will `set_injectable_page` be called with correct params?

Reply:
Perfect!